examples/vic-gov-au/nuxt.config.js

```diff
@@ -145,6 +145,10 @@ export default {
       alert: 1,
       gtm: 1
     },
+    profile: {
```

Comment:
I would like to move this object into the modules key - I think you should be able to pass a config object in (instead of just a 1 or 0) and we should just check for truthiness

Reply:
Agree. I am happy for you to do it in this PR or leave it in another ticket if it's too big to include in this PR(We then need to migrate this config later).